### PR TITLE
Upgrade parent and reformat using spotless

### PR DESCRIPTION
**Pull request by gnodet**
- #313
- Upgrade parent and reformat using spotless

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <groupId>org.codehaus.plexus</groupId>
     <artifactId>plexus-components</artifactId>
-    <version>10.0</version>
+    <version>14.2</version>
```
Comment (slachiewicz) [inline pom.xml:8]: 14.2 is now almost empty - so we can switch just to org.codehaus.plexus:plexus:15

Comment (gnodet) [reply pom.xml:8]: I've upgrade to latest released version afaik and 15 has not been released atm.

Comment (slachiewicz) [reply pom.xml:8]:
```suggestion
<parent>
    <groupId>org.codehaus.plexus</groupId>
    <artifactId>plexus</artifactId>
    <version>15</version>
  </parent>
```

Comment (gnodet) [reply pom.xml:8]: Ah, ok, I misunderstood what you said.  I'll have a look.

**Review by slachiewicz — APPROVED** [review]

**Pull request merged**